### DISCUSSION TITLE
posts/LLM%EA%B3%BC_%EC%82%AC%EA%B3%A0%EB%A0%A5%EC%97%90_%EB%8C%80%ED%95%9C_%EC%83%9D%EA%B0%81/

### DISCUSSION THREAD

**Discussion by giscus** [hobit22/hobit22.github.io#17] [General]
# posts/LLM%EA%B3%BC_%EC%82%AC%EA%B3%A0%EB%A0%A5%EC%97%90_%EB%8C%80%ED%95%9C_%EC%83%9D%EA%B0%81/

대 AI의 시대

https://hobit22.github.io/posts/LLM%EA%B3%BC_%EC%82%AC%EA%B3%A0%EB%A0%A5%EC%97%90_%EB%8C%80%ED%95%9C_%EC%83%9D%EA%B0%81/

**Comment by hobit22** [comment]
댓글 테스트 입니다 :)
